### PR TITLE
Update scaffold.tsx

### PR DESCRIPTION
**Pull request by remcohoff**

### CODE REVIEW

File: src/app/components/case/forms/CitizenReportForm/scaffold.tsx [hunk 1]
```diff
@@ -41,9 +41,10 @@ export default (caseId: Components.Schemas.Case["id"]) => {
           type: "TelField",
           props: {
             label: "Telefoonnummer melder",
+            extraLabel: "(indien bekend)",
             name: "reporter_phone",
             hint: "Vul hier alleen cijfers in",
-            isRequired: true,
+            isRequired: false,
             validate: (value: string) => (value === undefined || /^[0-9]{10}$/.test(value.trim())) ? false : "Vul hier enkel 10 cijfers in"
```
Comment (gilleswittenberg) [inline src/app/components/case/forms/CitizenReportForm/scaffold.tsx:48]: I think `/^[0-9]{10}$/.test(value.trim()) ? false : "Vul hier enkel 10 cijfers in"` would also work when `isRequired: true`

Comment (remcohoff) [reply src/app/components/case/forms/CitizenReportForm/scaffold.tsx:48]: The field is no longer required. But if someone enters a value, it has to be 10 characters long.

Comment (gilleswittenberg) [reply src/app/components/case/forms/CitizenReportForm/scaffold.tsx:48]: Ah, Ok. Read that wrong, then.

Comment (gilleswittenberg) [reply src/app/components/case/forms/CitizenReportForm/scaffold.tsx:48]: Maybe update the param type with undefined `validate: (value: string | undefined) => `

**Pull request merged**